src/components/toc/index.tsx

```diff
@@ -131,8 +131,8 @@ function TocEntry({ entry }: { entry: TocEntryType }) {
                 <Accordion open={openState}>
                     <ul
                         className={classNames({
-                            [`division-level-${entry.level}`]: entry.level,
-                        })}
+                            [`division-level-${entry.level}`]: entry.level
+                        }) + " structural"}
```

Comment:
`classNames` is magic. You don't need string concatenation. You can either do

```
classNames({ [`division-level-${entry.level}`]: entry.level }, "structural")
```
or

```
classNames({ [`division-level-${entry.level}`]: entry.level, structural: true })
```